pages/about.tsx

```diff
@@ -168,42 +202,73 @@ const AboutPage = () => {
 
       <SectionWrapper header={<H3>The return data</H3>} anchorKey="returndata">
         <p className="pb-8">
-          The return data region is the way a smart contract can return a value
-          after a call. It can be set by external contract calls through the{' '}
+          The return data is the way a smart contract can return a value after a
+          call. It can be set by external contract calls through the{' '}
```

Comment:
```suggestion
          call. It can be set by contract calls through the{' '}
```
First, it can be set by a call to itself, but most important, it sounds like you're using the solidity concept of "external", and it has nothing to do here, so I would rather not imply it.

Reply:
Done